src/as/math/Matrix3.ts

```diff
@@ -233,7 +235,8 @@ export class Matrix3 /*implements Matrix*/ {
 		return a * e * i - a * f * h - b * d * i + b * f * g + c * d * h - c * e * g
 	}
 
-	getInverse(matrix: Matrix3, throwOnDegenerate?: boolean): Matrix3 {
+	/** @returns false if the inverse can't be calculated (when the determinant is 0). In the case the inverse can't be calculated, the matrix is set to identity. */
```

Comment:
Exceptions (try-catch) are not yet supported in AssemblyScript, so instead we can return a boolean indicating whether it worked or not.